hello-world-apk/build.gradle

```diff
@@ -35,3 +35,10 @@ android {
         }
     }
 }
+
+dependencies {
+    //included just to test wildcard pom generation
+    compile group: 'org.apache.commons', name: 'commons-io', version: '1.3.+'
+    compile group: 'org.apache.commons', name: 'commons-io', version: '1.3.+'
```

Comment:
We should also test for a wildcard like 3.+, maybe replace one of these duplicate commons-io deps with everyone's favorite commons-lang3 to test that out?